.github/workflows/cicd.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       generate_release_package: true
 
   cmake:
-    if: github.event_name != 'pull_request'
+    #if: github.event_name != 'pull_request'
```

Comment:
why commented out?   Seems like it should be either present or absent, but not present and commented out